src/test/java/io/vertx/test/core/SocksProxy.java

```diff
@@ -30,7 +30,7 @@
 
   private static final Buffer clientInit = Buffer.buffer(new byte[] { 5, 1, 0 });
   private static final Buffer serverReply = Buffer.buffer(new byte[] { 5, 0 });
-  private static final Buffer clientRequest = Buffer.buffer(new byte[] { 5, 1, 0, 3 });
```

Comment:
The removed fourth byte signals the address type. 1 = IPv4, 3 = hostname. Additionaly, there is IPv6, but I would not know how to test that and apparently, it's not needed anyhow.